static/workflows/desktop-create-activity.json

```diff
@@ -0,0 +1,134 @@
+                    "action": "gcx:wf:core::If",
+                    "description": "custom ctivity pack registered with portal",
+                    "id": 8,
+                    "inputs": { "condition": true },
+                    "position": "-30,110",
+                    "title": "If",
+                    "transitions": [
+                        {
+                            "branch": "true",
+                            "id": 10,
+                            "position": "-30,165 -170,165 -170,230",
+                            "sourceConnector": "left",
+                            "target": { "id": 5 },
+                            "targetConnector": "top"
+                        },
+                        {
+                            "branch": "false",
+                            "id": 11,
+                            "position": "210,165 360,165 360,220",
+                            "sourceConnector": "right",
+                            "target": { "id": 7 },
+                            "targetConnector": "top"
+                        }
+                    ]
+                },
+                {
+                    "action": "gcx:wf:ui::Alert",
+                    "id": 13,
+                    "inputs": {
+                        "text": {
+                            "accessors": ["$myCustomActivityDirect"],
+                            "annotations": [
+                                { "count": 23, "index": 17, "kind": "idref" }
+                            ],
+                            "code": "\"Return value: \" + $myCustomActivityDirect.test",
+                            "source": "\"Return value: \"+$myCustomActivityDirect.test"
+                        },
+                        "title": "Directly called \"My custom actrivity\""
+                    },
+                    "position": "-290,370",
+                    "title": "Alert"
+                },
+                {
+                    "action": "gcx:wf:ui::Alert",
+                    "id": 20,
+                    "inputs": {
+                        "text": {
+                            "accessors": ["$runCustomActivity"],
+                            "annotations": [
+                                { "count": 18, "index": 17, "kind": "idref" }
+                            ],
+                            "code": "\"Return value: \" + $runCustomActivity[\"test\"]",
+                            "source": "\"Return value: \"+$runCustomActivity[\"test\"]"
+                        },
+                        "title": "Directly called \"My custom actrivity\""
+                    },
+                    "position": "240,370",
+                    "title": "Alert"
+                }
+            ]
+        }
+    ],
+    "deploymentConfig": { "supportedApps": { "VSD": true } },
+    "designerVersion": "5.35.0+38",
+    "licenseInfo": {
+        "licenseUrl": "https://vw-dev-qa-vgs.int.vertigis.com/vertigisstudio/workflow/service/auth/license"
```

Comment:
Please remove the `licenseInfo`. It'll get recreated by the designer when someone imports it.

Reply:
Removed